index.html

```diff
@@ -14,6 +14,56 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+      
+        <nav class="nav_link">
+            <img src="./img/karma-logo.svg" alt="">
+            <ul>
+                <li><a>Meet Karma</a></li>
+                <li><a>How it Works</a></li>
+                <li><a>Store</a></li>
+                <li><a>Blog</a></li>
+                <li><a>Help</a></li>
+                <li><a>Login</a></li>
+
+            </ul>
+        </nav>
+    </header>
+    <article class="main-container">
+        <h1>Introducing Karma</h1>
+        <p>Bring WIFI with you, everywhere you go.</p>
+        <button>Learn More</button>
+    </article>
+    <section class="container-services">
+        <p>Everyone nedds a little Karma</p>
+        <div class="container">
+        
+        <div class="services">
+            <img src="/img/icon-devices.svg" alt="">
+            <p> Internet for all devices </p>
+        </div>
+        <div  class="services">
+            <img src="/img/icon-coffee.svg" alt="">
+            <p> Boost your productivity </p>
+        </div>
+        <div  class="services">
+            <img src="/img/icon-refill.svg" alt="">
+            <p> Pay as You Go </p>
+        </div>
+    </div>
+        <hr>
+    </section>
+    <footer class="footer">
+        <div> <h5>Join us on</h5></div>
+        <div class="social-icon">
+                <a href="www.twitter.com"><img src="/img/twitter-icon.svg" alt=""></a>
+               <a href="www.facebook.com"><img src="/img/facebook-icon.svg" alt=""></a>
+                <a href="www.instagram.com"><img src="/img/instagram-icon.svg" alt=""></a>
+            
+            
+        </div>
+        <div><p>© Karma Mobility. Inc</p></div>
```

Comment:
```suggestion
        <p>© Karma Mobility. Inc</p>
```
The `div` is also not doing anything here so we can remove it